src/decawave/msg/Range.msg

```diff
@@ -3,5 +3,8 @@
 # http://wiki.ros.org/msg
 Header header
 string child_frame_id
+int32 id
 float64 distance
-float64 estimated_variance
+int32 distance_quality
+int32 quality_factor
+float64[] position
```

Comment:
This should probably be a geometry_msgs point3 or something
Should distance_quality or quality factor be a float, or express a standard deviation of trust?

Reply:
they are values given by the anchor for each distance reading. upon testing, they are allways 100, and do not seem useful. Not sure what they represent.